mrp_sale_info/models/sale_order_line.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Akretion (http://www.akretion.com).
+# @author Benoît GUILLOT <benoit.guillot@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    production_ids = fields.One2many(
+        comodel_name="mrp.production",
+        inverse_name="sale_line_id",
+        string="Manufacturing orders",
+    )
```

Comment:
It seems better to follow the field name in native module sale_mrp for sale orders

```suggestion
    mrp_production_ids = fields.One2many(
        comodel_name="mrp.production",
        inverse_name="sale_line_id",
        string="Manufacturing orders",
    )
```